### DISCUSSION TITLE
How can I next.js application on subpath of a domain?

### DISCUSSION THREAD

**Discussion by asynchronizd** [coollabsio/coolify#4000] [General]
I have to deploy nextjs application on a subpath. eg, domain.com/nextjs. How can I achieve it with coolify?

**Comment by redzovic** [comment]
Coolify: 
- In `General` set the `Domain` to your domain with the base path (e.g.: `example.com/docs`)
- Then go to `Advanced` and turn off `Strip Prefixes`

Next.js:
- Go to your `next.config.mjs` and add your base Path to the config under the property `basePath` 
(e.g.: `const config = {basePath: '/docs',...,};`)

(It might be that you also have to append `assetPrefix` to your config with the base path but I didn't test that as it work just fine for me right now)
